pyCGM_Single/pyCGM.py

```diff
@@ -660,71 +723,87 @@ def footJointCenter(frame,vsk,ankle_JC,knee_JC,delta):
     if foot flat is checked, make the reference markers instead of HEE marker which height is as same as TOE marker's height.
     elif foot flat is not checked, use the HEE marker for making Z axis.
 
-    -------------------------------------------------------------------
+    Markers used: RTOE, LTOE
+    Other landmarks used: ANKLE_FLEXION_AXIS
+    Subject Measurement values used: RightStaticRotOff, RightStaticPlantFlex, LeftStaticRotOff, LeftStaticPlantFlex
         
-    INPUT: dictionaries of marker lists.  
-            { [], [], [] }
-            An array of ankle_JC,knee_JC each x,y,z position.
-           delta = 0
-           static_info = [[R_plantar_static_angle, R_static_rotation_angle, 0], # Right Static information
-                          [L_plantar_static_angle, L_static_rotation_angle, 0]] # Left Static information
+    Parameters
+    ---------- 
+    frame : dict 
+        Dictionaries of marker lists.
+            { [], [], [], ... }
+    vsk : dict
+        A dictionary containing subject measurements from a VSK file.
+            { [], [], [], ... }
+    ankle_JC : array
+        An array of ankle_JC containing the x,y,z axes marker positions of the ankle joint center. 
+    knee_JC : array
+        An array of ankle_JC containing the x,y,z axes marker positions of the knee joint center. 
```

Comment:
Please be careful on copy-pasting things. It seems like you wrote a description for ankle_JC and copied to knee_JC